### PR TITLE
Add stale bot

### PR DESCRIPTION
**Pull request by kgpai**
This should use Github Actions Stale bot. The Github Actions Stale bot will mark stale PR's and Issues as 'stale' and eventually close them if no action takes place on them.

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -12,49 +12,25 @@
...
-
-# Limit to only `issues` or `pulls`
-only: pulls
+daysUntilClose: 14
+
+pulls:
+  daysUntilStale: 90 # ~3 months
+  markComment: >
+    This pull request has been automatically marked as stale because it has not had
+    recent activity. If you'd still like this PR merged, please comment on the task,
```
Comment (mbasmanova) [inline .github/stale.yml:22]: "comment on the task" doesn't make sense

File: .github/stale.yml [hunk 2]
```diff
@@ -12,49 +12,25 @@
...
+    make sure you've addressed reviewer comments, and rebase on the latest master.
```
Comment (mbasmanova) [inline .github/stale.yml:23]: master -> main

**Review by mbasmanova — COMMENTED** [review]
@kgpai Thank you for setting this up. Would you update PR description to describe what this is supposed to do?

File: .github/stale.yml [hunk 3]
```diff
@@ -12,49 +12,25 @@
...
+    Thank you for your contributions!
+
+issues:
+  daysUntilStale: 365 # 1 year
+  exemptLabels:
+    - bug
+  markComment: >
+    This issue has been automatically marked as stale because it has not had
+    any activity in the last year. If you feel that this issue is important,
+    just comment and the stale tag will be removed; otherwise it will be
```
Comment (majetideepak) [inline .github/stale.yml:33]: `just comment` -> `add a comment explaining why the issue is still relevant`

**Pull request closed without merging**